### PR TITLE
Test_Reports

### PR DESCRIPTION
**Pull request by sofathitesh**
Please accept this pull request.

### CODE REVIEW

**Comment by sofathitesh** [comment]
please check this pull request.

File: src/librehatti/Test_Reports/admin.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    form = Soil_building_Des
+    fields = ['Dt','Ob_Pr','Corr_F','Ob_N_V','Corr_N_V']
+    extra = 30
+    
+
+class Test_Reports_Admin(admin.ModelAdmin):
+    inlines = [Test_Reports_Inline]
+    model = Test_Reports
+
+
```
Comment (jasvir99) [inline src/librehatti/Test_Reports/admin.py]: remove extra endline.

File: src/librehatti/Test_Reports/forms.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+from django import forms
+
```
Comment (jasvir99) [inline src/librehatti/Test_Reports/forms.py]: remove endlines between imports.

File: src/librehatti/Test_Reports/forms.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+from librehatti.config import _SUB_CATEGORY
+
+from librehatti.config import _ITEM
+
+from librehatti.config import _TYPE
+
+
+class Test_Reports_Form(forms.ModelForm):
+
+
```
Comment (jasvir99) [inline src/librehatti/Test_Reports/forms.py]: remove extra endline.

File: src/librehatti/Test_Reports/migrations/0001_initial.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
```
Comment (jasvir99) [inline src/librehatti/Test_Reports/migrations/0001_initial.py]: no need to send migrations.

File: src/librehatti/Test_Reports/models.py [hunk 5]
```diff
@@ -0,0 +1,119 @@
+from django.db import models
+
+import useraccounts
```
Comment (jasvir99) [inline src/librehatti/Test_Reports/models.py]: remove extra endline from this file too.

File: src/librehatti/Test_Reports/models.py [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+from librehatti.config import _TEST_DATE
+
+from librehatti.voucher.models import FinancialSession
+
+from django.core.urlresolvers import reverse
+
+
+class Test_Reports(models.Model):
+    Session = models.ForeignKey(FinancialSession)
```
Comment (jasvir99) [inline src/librehatti/Test_Reports/models.py]: data fields are supposed to be in lower case.

File: src/librehatti/Test_Reports/models.py [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+    Col_N_V = models.CharField(max_length = 200, null=True, blank=True)
+    Col_Value = models.CharField(max_length = 200, null=True, blank=True)
+    Col_Net_V = models.CharField(max_length = 200, null=True, blank=True)
+    Col_G_V = models.CharField(max_length = 200, null=True, blank=True)
+
+    def __unicode__(self):
+        return '%s' % (self.Voucher) + ' - ' '%s' % (self.Date_of_Testing)
+
+class Soil_building_des(models.Model):
+    report_id = models.ForeignKey(Soil_building)
```
Comment (jasvir99) [inline src/librehatti/Test_Reports/models.py]: rename report_id to soil_building_report. No need to add id at the end.

File: src/librehatti/Test_Reports/urls.py [hunk 8]
```diff
@@ -0,0 +1,19 @@
+"""
+urls of catalog are..
```
Comment (jasvir99) [inline src/librehatti/Test_Reports/urls.py]: what is this?

File: src/librehatti/Test_Reports/views.py [hunk 9]
```diff
@@ -0,0 +1,167 @@
+from django.shortcuts import render
+
```
Comment (jasvir99) [inline src/librehatti/Test_Reports/views.py]: remove extra endlines.

File: src/librehatti/Test_Reports/views.py [hunk 10]
```diff
@@ -0,0 +1,167 @@
...
+import os
+
+from subprocess import call
+
+from .forms import Test_Reports
+
+
+@login_required
+def Test_Reports_data(request):
```
Comment (jasvir99) [inline src/librehatti/Test_Reports/views.py]: length of line should not be greater than 70 characters.

File: src/librehatti/config.py [hunk 11]
```diff
@@ -21,11 +21,19 @@
 """Account Information"""
 _ACCOUNT_HOLDER = 'ACCOUNT HOLDER'
```
Comment (jasvir99) [inline src/librehatti/config.py]: don't send your config file.

File: src/librehatti/programmeletter/admin.py [hunk 12]
```diff
@@ -4,8 +4,7 @@
 from librehatti.programmeletter.models import StaffInTeam
 from librehatti.programmeletter.models import LetterData
 
-from librehatti.programmeletter.forms import StaffInTeamForm
```
Comment (jasvir99) [inline src/librehatti/programmeletter/admin.py]: Please tell me the reason behind these changes.

File: src/librehatti/settings.py [hunk 13]
```diff
@@ -14,9 +14,9 @@
 DATABASES = {
     'default': {
```
Comment (jasvir99) [inline src/librehatti/settings.py]: we don't need your settings file.

File: static/js/ajax.js [hunk 14]
```diff
@@ -6,7 +6,7 @@ $(document).ready(function(){
     $('.parent_category').attr('disabled', 'disabled');
```
Comment (jasvir99) [inline static/js/ajax.js]: why these changes?

**Comment by jasvir99** [comment]
please check inline comments. 
We don't need to add logo as it is already present in images folder.
Why you have changed modes of files?
